pyproject.toml

```diff
@@ -19,7 +19,7 @@ exclude_lines = [
   "if TYPE_CHECKING:",
 ]
 ignore_errors = true
-fail_under = 89.0
+fail_under = 87.0
```

Comment:
Has coverage still dipped this low after you added unit tests?

Reply:
Actually the merge-keys PR accidentally removed some tests in `test_cli.py`. That's fixed now.

Reply:
Let's bump this up again then